app/src/main/java/com/ncnf/models/Organization.java

```diff
@@ -34,10 +34,6 @@ public Organization(String name, GeoPoint location, String address, String email
     }
 
     public Organization(UUID uuid, String name, GeoPoint location, String address, String email, String phoneNumber, List<String> adminIds, List<String> eventIds) {
-        if(checkCompleteList(adminIds) ){
-            throw new IllegalArgumentException("Organization should have at least one admin created when created");
-        }
-
```

Comment:
How do you check that a organization has at least oen administrator ?

Reply:
We don't. I don't think the model's responsibility is to check that condition: it simply models data from the database. Furthermore, I don't think this is a problem at this stage, because the organization is not created by the user but by the manager. So an organization can be created and the token simply pass to the first admin.